docs/90-FAQ.md

```diff
@@ -100,6 +100,8 @@ On zsh, any changes to $PATH you do in `~/.zshrc` will not be picked up (because
 this is only run for interactive shells). Use `~/.zshenv` instead to make
 additions to the path you want to use in awesome.
 
+If you need to execute command asynchronously (not blocking awesome while long command is running) you can use `awful.spawn.easy_async`.
```

Comment:
I think it makes more sense to introduce this with "If you need the output of a program". The stuff above (`spawn_with_shell`) is also async. :-)